src/frontend/src/flows/manage/index.ts

```diff
@@ -130,50 +152,57 @@ const style = () => html`<style>
 // recovery devices.
 const pageContent = (userNumber: bigint, devices: DeviceData[]) => html`
   ${style()}
-  <div class="container">
+  <section class="container">
     <h1>Anchor Management</h1>
     <p>
       You can view and manage this Identity Anchor and its added devices here.
     </p>
     ${!hasRecoveryDevice(devices) ? recoveryNag() : undefined}
-    <label>Identity Anchor</label>
-    <div class="highlightBox">${userNumber}</div>
-    <div class="labelWithAction">
-      <label class="labelAddedDevices"
-        ><span class="addedDevices">Added devices</span>
-        <span class="addedDevicesCounter"
-          ><span class="tooltip"
-            >You can register up to ${MAX_AUTHENTICATORS} authenticator devices
-            (recovery devices excluded)</span
-          >(${numAuthenticators(devices)}/${MAX_AUTHENTICATORS})</span
-        ></label
-      >
-      <button
-        ?disabled=${numAuthenticators(devices) >= MAX_AUTHENTICATORS}
-        class="labelAction"
-        id="addAdditionalDevice"
-      >
-        <span class="tooltip"
-          >You can register up to ${MAX_AUTHENTICATORS} authenticator devices.
-          Remove a device before you can add a new one.</span
+    <aside>
```

Comment:
Curious about the role of aside here, how it's different from role of an article? (I'm still quite new with semantics and would like to understand your approach!)

Reply:
An aside is used whenever an information does not make much sense on its own. In other words: You need the context (the article/section around) to understand its content.